### PR TITLE
List title

### PR DESCRIPTION
**Pull request by jrmneveu**
Fix the design side of the issue #35 Add a title to the list.

It remains 2 tasks : 

- Display a modal (like the one used in "Create a board") to give a name to the list
- Display the good name for each list

It appears a problem is coming. Today we already have some lists but they don't have any name. I fix the problem for the size in this case. 
But it can be a problem for the futur of the existing list, especially in the vertical gridBoard. Do we give them a default name ? Or do we ignore them ? Is it another issue ?

### CODE REVIEW

**Comment by asdine** [comment]
It's an issue that must be handle in this PR.
Will the lists always have a title?
- If yes, we need to gracefully display lists that don't have titles, as a practice.
We can give them a default title like `#1`, `#2`, etc. if it's easier.
- If no, the design must work for both cases.

**Comment by jrmneveu** [comment]
I think the idea to give them as title a number like you said is a good and simple way for the moment. Because as we want to give the opportunity to change the name of the list by clicking on it, if there is nothing it could be disturbing. 
We could also put a placeholder to indicate where to click, but let's do it simple for the moment.
So if it is easy for you, I am ok with giving them a default title like #1, #2...

File: desktop/app/scss/pulpe-structure/pulpe-list.scss [hunk 1]
```diff
@@ -17,6 +17,16 @@
 				margin-bottom: 0.625rem;
 			}
 
+			.plp-list-top {
```
Comment (asdine) [inline desktop/app/scss/pulpe-structure/pulpe-list.scss]: Should we use the same wording as bootstrap?
- `top` -> `header`
- `bottom` -> `footer`

Comment (jrmneveu) [inline desktop/app/scss/pulpe-structure/pulpe-list.scss]: Good question. I will work on this during my next css refactor. Can we keep going like this for this feature ?

Comment (asdine) [inline desktop/app/scss/pulpe-structure/pulpe-list.scss]: Sure :+1

**Comment by jrmneveu** [comment]
👍

**Pull request merged**